Cargo.toml

```diff
@@ -12,8 +12,8 @@ readme = "README.md"
 license = "MIT"
 
 [dependencies]
-bdk = { version = "^0.3", default-features = false }
-bdk-macros = "^0.2"
+bdk = { git = "https://github.com/bitcoindevkit/bdk.git", rev = "c4f2179", features = ["all-keys"]}
```

Comment:
Needs `default-features = false`, otherwise it'll include `electrum` by default which can't be compiled under `wasm32`

Reply:
Fixed. Had to tweek docs tests a bit but should all work now.